notes/force-move-spec.md

```diff
@@ -0,0 +1,208 @@
+# Force Move Spec
+
+To serve as a succinct specification of the ForceMove protocol.
+Everthing required for getting to an outcome.
+
+Things have changed since the original paper.
+Names have changed. Optimizations have been added.
+This should serve as the definitive version over time.
+
+Not a concrete specification.
+Abstract notion of "the state of the channel".
+Don't specify formats.
+In methods, we provide the minimal required parameters.
+In a concrete implementation, the parameters might change - for example it can be more efficient
+
+## Off-chain state
+
+| **Field**      | **Data type**                 | **Definition / Explanation**                                                                         |
+| :------------- | :---------------------------- | :--------------------------------------------------------------------------------------------------- |
+| ChainID        | `bytes32`                     | e.g. ropsten / mainnet                                                                               |
```

Comment:
I think this can be a uint256

Reply:
(Is there a difference though?)

Reply:
I guess not. Could also save space though because most chain IDs are pretty tiny https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md#list-of-chain-ids

Reply:
https://ethereum.stackexchange.com/questions/3067/why-does-uint8-cost-more-gas-than-uint256